assets/js/blocks/rating-filter/block.tsx

```diff
@@ -272,6 +273,19 @@ const RatingFilterBlock = ( {
 		[ checked ]
 	);
 
+	if ( isEditor && ! isDisabled && displayedOptions.length === 0 ) {
+		return (
+			<Notice status="warning" isDismissible={ false }>
+				<p>
+					{ __(
+						'There are no products with ratings.',
```

Comment:
I think we can be more explicit to merchants here by mentioning that the block won't display on the front end until their store has reviews.

Reply:
Yeah, that's a good call: in fact, I believe we could be more explicit about this on other blocks as well, such as the Filter Attributes. Already tagged @vivialice here earlier: looking forward to hearing her thoughts about this copy.